setupApp.py

```diff
@@ -94,7 +94,7 @@ def dyld_find(name, loader=None, **kwargs):
             'psychopy_ext', 'pyfilesec',
             'bidi', 'arabic_reshaper',  # for right-left language conversions
             # for Py3 compatibility
-            'future', 'past', 'lib2to3',
+            'lib2to3',
```

Comment:
@mdcutone was there a need to keep lib2to3? I'm not sure what that does at this point